spec/src/main/asciidoc/jsonb.adoc

```diff
@@ -318,7 +318,54 @@ Deserialization into anonymous classes is not supported. Serialization of anonym
 
 === Polymorphic Types
 
-Deserialization into polymorphic types is not supported by default mapping.
+Polymorphic type handling is supported for deserialization and serialization. Polymorphic handling is ensured by annotation `@JsonbPolymorphicType` and `@JsonbSubtype`. `@JsonbPolymorphicType` defines overall strategy how polymorphic information will be serialized/deserialized and defines all the supported aliases with using `@JsonbSubtype` annotations. `@JsonbSubtype` ensures proper and safe mapping between class alias and type. Implementation must validate mapped types if they are assignable from the annotated type. If not, an exception must be thrown.
```

Comment:
`@JsonbPolymorphicType` defines how the overall polymorphic information strategy will be serialized/deserialized and defines all the supported aliases using `@JsonbSubtype` annotations.